src/scripts/cmake/Sanitizers.cmake

```diff
@@ -0,0 +1,34 @@
+# Module to set flags for sanitizers
+
+# Inspired on http://www.stablecoder.ca/2018/02/01/analyzer-build-types.html
+
+if(CMAKE_BUILD_TYPE STREQUAL "Sanitizer")
+    set(Sanitizer_TYPE "" CACHE
+            STRING "Choose the sanitizer to use.")
+
+    set_property(CACHE Sanitizer_TYPE PROPERTY STRINGS
+            "" "TSAN" "ASAN" "LSAN" "MSAN" "UBSAN")
```

Comment:
Do we really need an empty flag? Once we enable the sanitizer build, it should default one of them.

Reply:
I will put ASAN as default